client/js/map.js

```diff
@@ -396,7 +396,7 @@ var initialize = function () {
         initialZoom = linkZoom;
     }
 
-    if (token != null || organization != null || treeid != null || userid != null || donor != null) {
+    if (token != null || organization != null || treeid != null || userid === null || donor != null) {
```

Comment:
Why did we change != to === here?  We don't want to set initial zoom to 10 when a userid is specified, as we do for other subsets?

Reply:
i don't think we do. not sure. with zoom there comes bounds, i believe. the map won't self center to an area where it finds trees, that's the fundamental problem. i am a bit lost in the spaghetti conditionals figuring out what all the intentions are there, but the behavior i was getting when having a userid, the map puts me to the middle of the pacific ocean, it doesn't self-center to an arbitrary area where the user actually has trees. not sure. any suggestions?